### PR TITLE
Fix to issue 9338

### LINKED ISSUE #9338: Utilize built in f32 min function instead of custom implementation
Replace

https://github.com/servo/servo/blob/dba1f27305c5e81eda6acd4c438a2adfb6ed053d/components/layout/display_list_builder.rs#L1931-L1937

with

https://doc.rust-lang.org/std/primitive.f32.html#method.min

### PR DESCRIPTION
**Pull request by njskalski**
Fixes #9338.

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9341)

### CODE REVIEW

**Comment by askalski** [comment]
maybe I should have left it to somebody else

**Comment by KiChjang** [comment]
@askalski Please try not to work on issues without claiming them first on GitHub. We had cases where two or more people were working on the same issue, and we had to throw away one or more people's work.

File: components/layout/display_list_builder.rs [hunk 1]
```diff
@@ -1983,4 +1975,3 @@ pub enum StackingContextCreationMode {
     OuterScrollWrapper,
     InnerScrollWrapper,
 }
-
```
Comment (KiChjang) [inline components/layout/display_list_builder.rs]: Revert this please.

**Comment by nox** [comment]
It would be nice to write proper commit messages btw. One day, GitHub will disappear and we won't know what's issue 9338.

**Comment by askalski** [comment]
KiChjang , nox: you are right. I apologize and focus on my task instead.

**Comment by KiChjang** [comment]
@bors-servo retry
* #9374

**Pull request merged**